src/utils/getCalendarMonthWeeks.js

```diff
@@ -1,6 +1,7 @@
 export default function getCalendarMonthWeeks(month, enableOutsideDays) {
-  const firstOfMonth = month.clone().startOf('month');
-  const lastOfMonth = month.clone().endOf('month');
+  const baseDate = month.clone().utcOffset(month.utcOffset());
```

Comment:
Wonderful! Can you add a comment here in the code as well, just to give a little bit of explanation?